bundle/jsky.app.ot/src/main/java/jsky/app/ot/viewer/SPElevationPlotPlugin.java

```diff
@@ -308,7 +328,13 @@ private LegendItemCollection getLegendItems() {
                         plic.add(new LegendItem(prioStr, p));
                     }
                     return plic;
-                }, obs -> String.format("%s Priority", ((SPObservation) obs.getDataObject()).getPriority().displayValue()));
+                }, (o1, o2) -> {
+                            final SPObservation.Priority p1 = ((SPObservation) o1.getDataObject()).getPriority();
+                            final SPObservation.Priority p2 = ((SPObservation) o2.getDataObject()).getPriority();
+                            return p1.compareTo(p2);
```

Comment:
`Enum` implements `Comparable` so could you use your `Comparator.comparing` trick here as well?  Something like `Comparator.comparing(o -> ((SPObservation) o.getDataObject()).getPriority)`?

Reply:
I'll give it a try. Thanks for looking at this PR.